assets/wizards/dashboard/views/DevInfo/style.scss

```diff
@@ -0,0 +1,62 @@
+@import '../../../../shared/scss/colors';
+@import '~@wordpress/base-styles/colors';
+
+.newspack-dashboard__news {
+	details {
+		margin-bottom: 5px;
+		border: 1px solid $light-gray-200;
+		padding: 20px;
+		border-radius: 3px;
+		> div {
+			border-radius: 2px;
+			margin: 20px 0 0;
+			padding-left: 40px;
+			h3:first-child {
+				margin-top: 0;
+			}
+		}
+		ul {
+			list-style: disc;
+			padding-left: 18px;
+		}
+		li {
+			margin-bottom: 0;
+		}
+		ul:last-child {
+			margin-bottom: 0;
+		}
+		h3 {
+			margin: 0;
+		}
+		svg {
+			transition: transform 200ms;
+		}
+		&[open] {
+			svg {
+				transform: rotate( 180deg );
+			}
+		}
+	}
+	summary {
+		display: flex;
+		font-size: 16px;
```

Comment:
I think it would be good to get rid of the larger font size, so these are more in line with text sizes on screens like 'Engagement' -- open to discuss this though!

```suggestion
```

Reply:
Removed in e2cd04c